sale_stock_ux/data/ir_actions_data.xml

```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <record id="sale_order_massive_cancel" model="ir.actions.server">
+        <field name="name">Masive Canceltion for remaining stock</field>
+        <field name="type">ir.actions.server</field>
+        <field name="model_id" ref="model_sale_order"/>
+        <field name="binding_model_id" ref="model_sale_order"/>
+        <field name="state">code</field>
+        <field name="code">
+action = {
+    'type': 'ir.actions.act_window.message',
+    'title': 'Masive Canceltion for remaining stock',
+    'close_button_title': False,
+    'message': 'Do You want to cancel remaining for this orders?',
```

Comment:
Poner "Do you want to cancel remaining quantities for this orders? This can't be undone."